configure.ac

```diff
@@ -5064,6 +5064,12 @@ AS_CASE([$FIPS_VERSION],
         AS_IF([test "x$ENABLED_DES3" = "xno"],[ENABLED_DES3="yes"])
     ])
 
+AS_IF([test "x$ENABLED_FIPS" = "xyes" && test "x$ENABLED_ALL" = "xyes"],
+    [AC_MSG_ERROR([FIPS is incompatible with --enable-all])])
+
+AS_IF([test "x$ENABLED_FIPS" = "xyes" && test "x$ENABLED_ALL_CRYPT" = "xyes"],
+    [AC_MSG_ERROR([FIPS is incompatible with --enable-all-crypto])])
+
```

Comment:
We use `--enable-fips=dev` in combination with `--enable-all` and `--enable-all-crypto` to test new/future features.